apk-generator/v2/app/generator/generator.py

```diff
@@ -327,24 +329,37 @@ def run_command(self, command):
         rc = process.poll()
         return rc
 
+    @socketio.on('message')
+    def handle_message(message):
+        send(message)
+
     def generate_status_updates(self, output_line):
         if 'Starting process \'Gradle build daemon\'' in output_line:
+            handle_message('Starting gradle builder')
             self.update_status('Starting gradle builder', skip_log=True)
```

Comment:
I have not looked into other files, but if there is no reference of update_status without the handle_message call, why not include it in the function itself?